appendix/migrate-to-postgresql.rst

```diff
@@ -117,70 +69,105 @@ Install pgloader and migrate
          $ zypper install pgloader
 
 
+Create pgloader command file
+----------------------------
+
+Create a command file for pgloader with:
+
+.. tabs::
 
-Create command file
--------------------
+   .. tab:: Package installation
 
-Create a command file for pgloader for example in ``/tmp/pgloader-command``:
+   .. code-block:: sh
 
-.. code-block:: cfg
+      $ zammad run rake zammad:db:pgloader > /tmp/pgloader-command
 
+   .. tab:: Source installation
 
-   LOAD DATABASE
-		FROM mysql://zammad:mysql_password@localhost/zammad
-		INTO pgsql://zammad:pgsql_password@localhost/zammad
+      .. code-block:: sh
 
-   ALTER SCHEMA 'zammad' RENAME TO 'public'
+         $ su - zammad
+         $ rake zammad:db:pgloader > /tmp/pgloader-command
 
-   WITH BATCH CONCURRENCY = 1;
 
+Afterwards you need to align the created file with the needed information.
```

Comment:
@dvuckovic it might be good to show an example here for what needs to be changed. I guess credentials have to be changed almost always for production instances.

Reply:
There is already a helpful comment in the command file just above the line with the target PostgreSQL URL which has to be tweaked:

```
LOAD DATABASE
  FROM mysql://root:zammad@127.0.0.1/zammad_development

  -- Adjust the PostgreSQL URL below to correct value before executing this command file.
  INTO pgsql://zammad:zammad@localhost/zammad_development

ALTER SCHEMA 'zammad_development' RENAME TO 'public'
...
```

The rest of the values in the file should be current, they are automatically read from the database configuration of the current environment.

I will add the same info in the PR.